### DISCUSSION TITLE
Github workflow status badges - custom left-hand-side text!

### DISCUSSION THREAD

**Discussion by TheSquidCombatant** [community/community#63964] [Actions]
### Select Topic Area

Product Feedback

### Body

Today i could not change badge left-hand-side text, but i wanna.

I mean https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge

I suggest to add support for the URL parameter "label" as done in https://shields.io/badges/git-hub-tag-checks-state

[Ticket 2292801](https://support.github.com/ticket/2292801).

**Comment by nefeleai** [comment]
Did this ticket get any attention? Is sad to see people running out to 3rd parties just for not having a desired label

**Reply by pixelpicnic** [reply]
This would be very helpful when specifying ?branch=BRANCH_NAME.
The left side of the badge should reflect the branch being tested
